### DISCUSSION TITLE
Lighting Effect (Stadium light)

### DISCUSSION THREAD

**Discussion by Chicknvictim** [T-Troll/alienfx-tools#402] [Ideas]
it would be great if we add stadium light changes acc to the color of the screen , which would give us immersive experience when we play games , give that ambient mode especially in color toned games .

**Comment by T-Troll** [comment]
It's implemented already! ^_^
Check "Ambient" effect.

**Reply by Chicknvictim** [reply]
Wait it's not showing for Alienware laptop, mine is x17 r2 and there isn't any option for ambient mode

**Reply by T-Troll** [reply]
![image](https://github.com/T-Troll/alienfx-tools/assets/51940412/3c9f4ffe-b0a0-4007-acfc-5941daf07bfa)
